devnet/query-server.yaml

```diff
@@ -56,6 +56,8 @@ spec:
             - --bootstrap
             - /dns4/guardian-0.guardian/udp/8996/quic/p2p/12D3KooWL3XJ9EMCyZvmmGXL2LMiVBtrVa2BuESsJiXkSj7333Jw
             - --logLevel=info
+            - --shutdownDelay1
```

Comment:
do you need to set a value for shutdownDelay2?

Reply:
Don't need to. If delay1 is zero, it never checks delay2.

Reply:
If shutdownDelay1 == 0, shutdownDelay2 isn't used, correct?